src/styles/card.scss

```
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 42px;
+  }
+
+  &__name {
+    margin: 0;
+    margin-bottom: 5px;
```

I find it very bad practice. first you set margin-bottom to 0, then override it to 5px in next line. You can use margin shorthand here:
```suggestion
    margin: 0 0 5px;
```

---

also i recommend reading about logical margins (and other props): https://css-tricks.com/css-logical-properties-and-values/